src/components/PageFooter.vue

```diff
@@ -37,22 +43,49 @@
                 </div>
                 <div class="col-12 col-sm-6 col-md-4">
                     <h4>Find us</h4>
-                    <a href="https://github.com/aetheric-oss" target="_blank"
-                        >Visit our Github</a
-                    ><br />
+                    <br />
+                    <div id="footer-icon-bar">
+                        <a href="https://github.com/aetheric-oss" target="_blank">
+                            <Icon class="footer-icon" icon="hugeicons:github-01" />
+                        </a>
+                        <a href="https://www.linkedin.com/company/aetheric-bv" target="_blank">
+                            <Icon class="footer-icon" icon="hugeicons:linkedin-02" />
+                        </a>
+                        <a href="https://bsky.app/profile/aetheric-oss.bsky.social" target="_blank">
+                            <Icon class="footer-icon" icon="hugeicons:bluesky" />
+                        </a>
+                    </div>
+                    <br />
                 </div>
             </div>
         </div>
     </div>
-    <footer
-        class="container-fluid py-2 bg-dark text-light"
-    >
+    <footer class="container-fluid py-2 bg-dark text-light">
         <div class="container-xl">
             <div class="row">
-                <div class="col-12">
-                    © Copyright 2023 - 2024 • Aetheric • kvk 93258798
-                </div>
+                <div class="col-12">© Copyright 2023 - 2025 • Aetheric BV • KVK 93258798</div>
             </div>
         </div>
     </footer>
 </template>
+
+<style scoped>
+.row {
```

Comment:
Bootstrap provides the classes:
- text-center https://getbootstrap.com/docs/5.3/utilities/text/
- justify-content https://getbootstrap.com/docs/5.3/utilities/flex/#justify-content

You should only need 1 of these to center the content inside an element.
justify-items doesn't exist in flexbox https://stackoverflow.com/questions/32551291/in-css-flexbox-why-are-there-no-justify-items-and-justify-self-properties